test/e2e/framework.go

```diff
@@ -86,7 +86,7 @@ const (
 	busyboxImage        = "projects.registry.vmware.com/library/busybox"
 	nginxImage          = "projects.registry.vmware.com/antrea/nginx"
 	perftoolImage       = "projects.registry.vmware.com/antrea/perftool"
-	ipfixCollectorImage = "projects.registry.vmware.com/antrea/ipfix-collector:10282020.1"
+	ipfixCollectorImage = "projects.registry.vmware.com/antrea/ipfix-collector:latest"
```

Comment:
Ideally we should move to an image with tag as latest may keep changing. When we bump up go-ipfix version to 0.3.1, we should add the tag.

Reply:
Sure. Will modify it in go-ipfix version bump PR. Thanks!